server/src/database/schema.prisma

```diff
@@ -886,3 +887,23 @@ model ViewField {
   @@id([viewId, key])
   @@map("viewFields")
 }
+
+model ApiKey {
+  /// @Validator.IsString()
+  /// @Validator.IsOptional()
+  id String @id @default(uuid())
+  name String
+  /// @TypeGraphQL.omit(input: true, output: false)
+  key String @unique
+  /// @TypeGraphQL.omit(input: true, output: true)
+  workspace   Workspace @relation(fields: [workspaceId], references: [id])
+  /// @TypeGraphQL.omit(input: true, output: true)
+  workspaceId String
+  expiresAt DateTime?
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  /// @TypeGraphQL.omit(input: true, output: true)
+  deletedAt DateTime?
```

Comment:
@martmull You need to add a `revokedAt` key when we don't store the token

Reply:
ok